cmd/gitops/beta/run/cmd.go

```diff
@@ -373,13 +378,18 @@ func dashboardStep(ctx context.Context, log logger.Logger, kubeClient *kube.Kube
 
 			// Answering "n" causes err to not be nil. Hitting enter without typing
 			// does not return the default.
-			_, err := prompt.Run()
+			answer, err := prompt.Run()
 			if err == nil {
-				wantToInstallTheDashboard = true
+				wantToInstallTheDashboard = wantToInstall
+			} else if answer == "n" || answer == "N" {
+				wantToInstallTheDashboard = doNotWantToInstall
+				flags.SkipDashboardInstall = true
+			} else {
+				wantToInstallTheDashboard = unknown
 			}
```

Comment:
If I'm not mistaken this is the only case where `wantToInstallTheDashboard` would be set to `unknown` (which it already is, anyway), so this assignment is unnecessary.

If we decide to fail for any answer other than "n" or "N" then we could completely get rid of the `unknown` state and make `wantToInstallTheDashboard` a bool.

wdyt?

Reply:
Thinking about it, we don't need the `unknown` state, anyway because further down we only evaluate `wantToInstallTheDashboard == wantToInstall`.

Reply:
Yep. That's an over design. Converted back to a Boolean and it's still semantically correct.